scripts/visualize/visualize-doy.R

```diff
@@ -49,7 +51,14 @@ visualize.doy <- function(viz = getContentInfo(viz.id = "doy-NM")){
     xml_attr(polyline, "style") <- NULL
     xml_add_child(g.doys, polyline[[1]])
   }
-
+  
+  g.zoomer <- xml_add_sibling(xml_children(doy_svg)[[length(xml_children(doy_svg))]], 'g', id='total_hydro','class'='total_hydro')
+  xml_attr(g.zoomer, "transform") <- "translate(0,370)"
+  
+  xml_add_child(g.zoomer, zoomer.xml)
+  
+  xml_attr(doy_svg, "viewBox") <- paste(c(0,0,360,566), collapse=' ')
```

Comment:
in the future we might want to make this more robust to us changing the size of the other svg